Editorial changes to Retiring Datasets section (#65)
ACDguide/Governance: Governance/retire/retire-principles.md

It may be necessary to retire a dataset for several reasons:

- the data is seriously flawed and access to it needs to be removed
- the data is not used by our community anymore
- need to prioritise how storage is being used

> In any case we must proceed carefully and make sure we have considered all the implications of removing the data. As mentioned in the introduction, the most important factor is if the data is published.

Reviewer: If you're trying to remove the first person/make it passive, there's also a 'we' in the first sentence here. 
```suggestion
In any case it is vital proceed carefully and consider all the implications of removing the data. As mentioned in the introduction, the most important factor is if the data is published.
```
